book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+
```

Comment:
- [ ] You should `initialized` the rentals as it make accessible throughout the instance of the class in which it is defined Eg:
``@rentals = []``